recipes/llama-cpp/all/conanfile.py

```diff
@@ -83,14 +64,18 @@ def generate(self):
         deps.generate()
 
         tc = CMakeToolchain(self)
+        tc.variables["BUILD_SHARED_LIBS"] = bool(self.options.shared)
         tc.variables["LLAMA_STANDALONE"] = False
         tc.variables["LLAMA_BUILD_TESTS"] = False
         tc.variables["LLAMA_BUILD_EXAMPLES"] = self.options.get_safe("with_examples")
         tc.variables["LLAMA_CURL"] = self.options.get_safe("with_curl")
-        tc.variables["BUILD_SHARED_LIBS"] = bool(self.options.shared)
-        tc.variables["GGML_CUDA"] = self.options.get_safe("with_cuda")
-        if hasattr(self, "settings_build") and cross_building(self):
+        if cross_building(self):
             tc.variables["LLAMA_NATIVE"] = False
+            tc.variables["GGML_NATIVE_DEFAULT"] = False
+
+        tc.variables["GGML_BUILD_TESTS"] = False
+        #tc.variables["GGML_BUILD_EXAMPLES"] = self.options.get_safe("with_examples")
```

Comment:
Is this a leftover?

Reply:
More of a reminder that this currently breaks the build for examples for some reason, wanted to re-check today, but will remove it if I make no progress